src/current/_includes/releases/v24.1/v24.1.0-alpha.4.md

```diff
@@ -0,0 +1,116 @@
+- The `FORCE_INVERTED_INDEX` hint is now supported. This makes the [optimizer](../v24.1/cost-based-optimizer.html) prefer a query plan scan over any inverted index of the hinted table. The query will result in an error if no such query plan can be generated. [#120384][#120384]
+- The `REPAIRCLUSTERMETADATA` privilege now has an alternative name: `REPAIRCLUSTER`. Both names can be used interchangeably. [#116844][#116844]
+
+<h3 id="v24-1-0-alpha-4-operational-changes">Operational changes</h3>
+
+- The [cluster setting](../v24.1/cluster-settings.html) `admission.wal.failover.unlimited_tokens.enabled` can be set to `true` to cause unlimited admission tokens during write-ahead log (WAL) failover. This should not be changed without significant expertise, since the default, which preserves the token counts from the preceding non-WAL-failover interval, is expected to be safer. [#120135][#120135]
+- Setting the new [`cockroach start`](../v24.1/cockroach-start.html) option `--wal-failover=among-stores` or `COCKROACH_WAL_FAILOVER=among-stores` environment variable will configure a multi-store CockroachDB node to failover a store's write-ahead log (WAL) to another store's data directory. Failing over the write-ahead log may allow some operations against a store to continue to complete despite temporary unavailability of the underlying storage.. [#120509][#120509]
+- Introduced a new `storage.wal_failover.unhealthy_op_threshold` [cluster setting](../v24.1/cluster-settings.html) for configuring the latency threshold at which a WAL write is considered unhealthy. [#120509][#120509]
+- Added two new metrics to track the status of the SQL activity update job, which is used to pre-aggregate top K information within the SQL stats subsytem and write the results to `system.statement_activity` and `system.transaction_activity`:
+	- `sql.stats.activity.updates.successful`: Number of successful updates made by the SQL activity updater job.
+	- `sql.stats.activity.update.latency`: The latency of updates made by the SQL activity updater job. Includes failed update attempts. [#120522][#120522]
+- Added a new counter metric, `sql.stats.flush.done_signals.ignored`, that tracks the number of times the SQL activity update job ignored the signal indicating a flush has completed. This may indicate that the SQL activity update job is taking longer than expected to complete. [#120522][#120522]
+- Added a new counter metric, `sql.stats.activity.updates.failed`, to measure the number of update attempts made by the SQL activity update job that failed with errors. The SQL activity update job is used to pre-aggregate top K information within the SQL stats subsystem and write the results to `system.statement_activity` and `system.transaction_activity`. [#120522][#120522]
```

Comment:
```suggestion
- The new counter metric `sql.stats.activity.updates.failed` measures the number of times update attempts the SQL Activity Update job has attempted which have failed with errors. The SQL activity update job pre-aggregates top K information within the SQL Statistics subsystem and writes the results to `system.statement_activity` and `system.transaction_activity`. [#120522][#120522]
```
Consider combining with the note above, which relates to the same PR and has a redundant definition of the SQL Activity Update job.

Reply:
Good catch -- I'm just going to cut the second sentence.